packages/blip/app/components/patient-data.js

```diff
@@ -244,12 +237,34 @@ class PatientDataPage extends React.Component {
       break;
     }
 
+    let start = "1970-01-01";
+    let end = "1970-01-02";
+    if (canPrint && showPDFPrintOptions) {
+      const { startDate, endDate } = tidelineData.getLocaleTimeEndpoints();
+      start = startDate.format("YYYY-MM-DD");
+      end = endDate.format("YYYY-MM-DD");
+    }
+
+    const classes = clsx(
+      "patient-data",
+      "patient-data-yourloops",
+      { ["patient-data-with-dialog"]: showPDFPrintOptions }
+    );
+
     return (
-      <div className="patient-data patient-data-yourloops">
+      <div className={classes}>
         {messages}
         {patientData}
         {loader}
         {errorDisplay}
+        {canPrint &&
```

Comment:
to follow up on one of my comments on `pdf-print-options.tsx`:

```
{canPrint && showPDFPrintOptions &&
          <DialogPDFOptions
            minDate={start}
            maxDate={end}
            onResult={this.handlePrint}
          />
        }
```

Reply:
nope, we want the animation when showing the dialog. It should be left at is.

Reply:
Ok. I understand. I got confused by `showPDFPrintOptions` who's naming suggest that it is a boolean. I suggest renaming it to `pdfPrintOptions` or something less confusing.